Source/Core/AbsyCmd.cs

```diff
@@ -3775,6 +3781,15 @@ public MiningStrategy ErrorDataEnhanced
       set { errorDataEnhanced = value; }
     }
 
+    public AssertCmd(IToken /*!*/ tok, Expr /*!*/ expr, ProofObligationDescription description, QKeyValue kv)
```

Comment:
Can we reduce duplication by letting another constructor call this one with a `new AssertionDescription()` description?

Reply:
Can you reduce it further by setting the default value kv = null, since this is what you do in most places?